test/e2e/kubernetes/pod/pod.go

```diff
@@ -354,10 +350,7 @@ func Get(podName, namespace string, retries int) (*Pod, error) {
 	for i := 0; i < retries; i++ {
 		cmd := exec.Command("k", "get", "pods", podName, "-n", namespace, "-o", "json")
 		out, err = cmd.CombinedOutput()
-		if err != nil {
```

Comment:
These `kubectl get` errors are expected (thus the retries are built in), we don't need to print every one of them in-line in this function; rather we'll return the error to the consumer, who can decide to print out the equivalent error.